packages/core/src/Modules/AccountDeactivated/account-deactivated.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { getDerivComLink } from '@deriv/shared';
+
+const AccountDeactivated = ({ logout }) => {
```

Comment:
Is there a reason this component is in Core instead of Account? It breaks decoupling of logic. The logic to deactivate your account is in Account, thus the success modal should be too.

Reply:
I'll move it to the Account package

Reply:
since we are rendering the `vertical_tab` component in the Account package we can't add this `account_deactivated` here based on the design (we should not show vertical_tab here).
so I need to add it in core and render it there.

Reply:
ok, moved it to the account package.